src/node/rpc/memberfrontend.h

```diff
@@ -411,6 +419,8 @@ namespace ccf
     NetworkTables& network;
     AbstractNodeState& node;
     const lua::TxScriptRunner tsr;
+    // For now, shares are stored in enclave memory
```

Comment:
I don't think we're going to worry about swapping them out in the foreseeable future.

Reply:
Bad phrasing from me: I meant that the submitted shares will be stored in the KV soon. I'll rephrase.